src/ATen/function_wrapper.py

```diff
@@ -215,12 +215,30 @@ def to_return_type(arg, option):
         if not is_mutable_formal_argument(arg, option):
             rt = 'const ' + rt
     return {
+        'name': arg['name'],
         'type': rt,
         'dynamic_type': DYNAMIC_TYPE.get(arg['type'], arg['type']),
     }
 
 
 def create_generic(top_env, declarations):
+    # translates defaults from cwrap types to C++ values
+    def translate_default(argument, type_str, default):
+        if default is None:
+            return '{}'
```

Comment:
a comment would be nice:
// cause the default constructor for the object to run